internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -345,17 +345,16 @@ func ParseSessionCallAttemptComplete(c SessionCallAttempt) (d sdktypes.SessionCa
 
 type Session struct {
 	SessionID        string `gorm:"primaryKey"`
-	DeploymentID     string `gorm:"index;foreignKey"`
+	BuildID          string `gorm:"index"`
+	EnvID            string `gorm:"index"`
+	DeploymentID     string `gorm:"index"`
 	EventID          string `gorm:"index"`
 	CurrentStateType int    `gorm:"index"`
 	Entrypoint       string
 	Inputs           datatypes.JSON
 	CreatedAt        time.Time      //`gorm:"default:current_timestamp"`
 	UpdatedAt        time.Time      //`gorm:"default:current_timestamp"`
 	DeletedAt        gorm.DeletedAt `gorm:"index"`
```

Comment:
let's add a Note/TODO/FIXME regarding what should be foreign key and what should be a constraint?

Reply:
done